push-notifications/android/src/main/java/com/capacitorjs/plugins/pushnotifications/PushNotificationsPlugin.java

```diff
@@ -128,13 +129,18 @@ public void getDeliveredNotifications(PluginCall call) {
     public void removeDeliveredNotifications(PluginCall call) {
         JSArray notifications = call.getArray("notifications");
 
-        List<Integer> ids = new ArrayList<>();
         try {
             for (Object o : notifications.toList()) {
                 if (o instanceof JSONObject) {
                     JSObject notif = JSObject.fromJSONObject((JSONObject) o);
-                    Integer id = notif.getInteger("id");
-                    ids.add(id);
+                    String tag = notif.getString("tag");
+                    Integer id = notif.getInteger("id", 0);
```

Comment:
The id is not optional, so you shouldn't need the default value. 
Have you seen it being null?

Reply:
not really, I removed it.